checks/check-app-health.go

```diff
@@ -0,0 +1,59 @@
+package checks
+
+import (
+	"fmt"
+	"strings"
+	"time"
+	"net/http"
+	"io/ioutil"
+    "strconv"
+    "log"
+
+	maps "github.com/ashwanthkumar/golang-utils/maps"
+	"github.com/gambol99/go-marathon"
+)
+
+type AppHealth struct{}
+
+func (h *AppHealth) Name() string {
+	return "apphealth"
+}
+
+// Checks App health status of App Endpoint by reading router.hosts 
+//  and Check for 200 status code
+func (h *AppHealth) Check(app marathon.Application) AppCheck {
+	host := maps.GetString(app.Labels, "router.hosts", "notDefined")
```

Comment:
Also internally this `router.hosts` can also contain multiple dns names. So you might want to account for that as well.

Reply:
Accounted for multiple router.hosts in this commit and it now checks the first dns name for HTTP check. I don't believe there is any need to check other hosts which will be taken care of in the dns check https://github.com/ashwanthkumar/marathon-alerts/pull/35/commits/9d95e851bec7f1fcd97f3a2a2bdba3c4f42bcc44